tests/memcached/memcached.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load ../utils
+
+
+@test "Start memcached server" {
+    sudo docker run --name memcached-server --rm -d clearlinux/memcached
+    sleep 5
+    run check_container_status memcached-server
+    [ "$status" -eq 0 ]
+}
+
+@test "Accessing statistics" {
+    # get ip address
+    run get_container_ip memcached-server
+    [ "$status" -eq 0 ]
+    ipaddr="$output"
+
+    echo stats | nc $ipaddr 11211
+}
+
+
+@test "Stop memcached server" {
+    sudo docker stop memcached-server
+    sleep 3
+    run check_container_status memcached-server
+    [ "$status" -eq 1 ]
+}
```

Comment:
I'd prefer tests that will fail if an earlier test would fail would just be a single test. This can allow later tests to run in parallel at some point in cases where we might start two containers at once for testing and use different ports for instance.

Reply:
No sorry I mean, make these 3 tests a single test in cases where they are dependent and this goes for other tests. This would make it easier to enable running tests in parallel as each test would start and stop their own docker container.

Reply:
Ok, got you. So for the ease of adding other tests, I will add setup/teardown for them.

Reply:
They don't need to be bats setup and teardown methods for all cases even. Having bash functions that are called by the tests to do create and delete images (likely with a name parameter at least for the container) makes complete sense.

Reply:
The container starting parameters, commands, even the wait-for stable time may be different.
I 'd ratter keep this start/stop inside each test cases.

Reply:
That's reasonable to me.